presto-prometheus/src/main/java/io/prestosql/plugin/prometheus/PrometheusRecordCursor.java

```diff
@@ -139,8 +141,10 @@ public long getLong(int field)
     {
         Type type = getType(field);
         if (type.equals(TIMESTAMP_COLUMN_TYPE)) {
-            Timestamp timestamp = (Timestamp) requireNonNull(getFieldValue(field));
-            return timestamp.toInstant().toEpochMilli();
+            Instant dateTime = (Instant) requireNonNull(getFieldValue(field));
+            // render with the fixed offset of the Presto server
```

Comment:
This seems strange the data is coming from a different system (unlikely JMX where it's local to that JVM). Using session time zone seems better. Though in practice these are likely to be the same.

Reply:
I think it should either be JVM or a configuration.  We can add a configuration later if people want.